node-module-system/calculate.js

```diff
@@ -0,0 +1,16 @@
+import add from './add.js';
+import subtract from './subtract.js';
+import multiply from './multiply.js';
+import divide from './divide.js';
+
+const addResult = add(2, 3);
```

Comment:
This code isn't reading the command line or switching on the operation. When you get a chance, you'll want to finish this.